src/porepy/models/solution_strategy.py

```diff
@@ -451,12 +460,24 @@ def after_nonlinear_convergence(self) -> None:
         self.save_data_time_step()
 
     def after_nonlinear_failure(self) -> None:
-        """Method to be called if the non-linear solver fails to converge."""
+        """Method to be called if the non-linear solver fails to converge.
+
+        Parameters:
```

Comment:
Should this have been added to the method signature?

Reply:
Good catch! That was an artefact of my intermediate code, and it's not used. I'll just remove it from the signature.